jsonpb/jsonpb.go

```diff
@@ -973,13 +973,6 @@ func (u *Unmarshaler) unmarshalValue(target reflect.Value, inputValue json.RawMe
 		}
 		if mp != nil {
 			target.Set(reflect.MakeMap(targetType))
-			var keyprop, valprop *proto.Properties
-			if prop != nil {
-				// These could still be nil if the protobuf metadata is broken somehow.
-				// TODO: This won't work because the fields are unexported.
```

Comment:
Please keep remove the TODO comment. It's still a valid work to do.

Reply:
Done (reworded so it makes sense now that the anchoring code is gone).

Reply:
Note that the new comment follows the example of several other TODOs in this file.